File: csharp/LeetTests/Old/ReverseBitsTest.cs
using Old.ReverseBits;

namespace LeetTests.Old
{
    internal class ReverseBitsTest
    {
        [Test]
        public void TestsReverseBits()
        {
            var solution = new Solution();
            Assert.That(
                solution.reverseBits(0b00000010100101000001111010011100),
                Is.EqualTo(0b00111001011110000010100101000000)
            );
            Assert.That(
                solution.reverseBits(0b11111111111111111111111111111101),
                Is.EqualTo(0b10111111111111111111111111111111)
            );
        }
    }
}

File: returned/python/today15/reverse_bits_test.py
from reverse_bits import Solution

def test1():
    assert Solution().reverseBits(0b00000010100101000001111010011100) == 0b00111001011110000010100101000000

def test2():
    assert Solution().reverseBits(0b11111111111111111111111111111101) == 0b10111111111111111111111111111111